src/Swift.Bindings/src/Parser/SwiftABIParser.cs

```diff
@@ -317,14 +332,16 @@ private List<BaseDecl> CollectDeclarations(IEnumerable<Node> nodes, BaseDecl par
         /// <returns>The struct declaration.</returns>
         private StructDecl CreateStructDecl(Node node, BaseDecl parentDecl, ModuleDecl moduleDecl, bool hasFrozenAttribute)
         {
+            var fullyQualifiedName = ExtractFullyQualifiedName(parentDecl.FullyQualifiedName, node.Name);
```

Comment:
What is the motivation behind moving the `ExtractFullyQualifiedName` to a separate variable assignment instead of doing it the previous way (inside the `new StructDecl`)?

Reply:
It is used in two places inside of new StructDecl so I hoisted it to avoid double computation